README.md

```diff
@@ -1,52 +1,74 @@
-
-## GSQ concepts
-1. Seismic Survey Lines (the lines of the seismic survey)
+Classes used in this profile:
+1. [GSQ Site](https://github.com/geological-survey-of-queensland/gsq-site-profile)
+2. [GSQ Dataset](https://github.com/geological-survey-of-queensland/gsq-dataset-profile)
+3. [GSQ Permit](https://github.com/geological-survey-of-queensland/gsq-permit-profile)
 
 ## OWL classes
 1. [SOSA sampling](https://www.w3.org/TR/vocab-ssn/#SOSASampling)
-2. [SOSA feature of interest](https://www.w3.org/TR/vocab-ssn/#SOSAFeatureOfInterest)
-3. [FOAF document](http://xmlns.com/foaf/spec/#term_Document) - used to capture the secondary metadata in JSON format
-4. [Geometry](https://www.w3.org/2003/01/geo/)
-5. [ProperInterval](https://www.w3.org/TR/owl-time/#time:ProperInterval)
+2. [dct:title](https://w3c.github.io/dxwg/dcat/#Property:resource_title) - report title
+3. [dct:description](https://w3c.github.io/dxwg/dcat/#Property:resource_description)
+4. [dct:identifier](https://w3c.github.io/dxwg/dcat/#Property:resource_identifier) - report number 
+5. xxx:alias
+6. [dct:created](https://w3c.github.io/dxwg/dcat/)
+7. [dct:issued](https://w3c.github.io/dxwg/dcat/#Property:resource_release_date) - date of formal issuance (e.g., open file publication)
+8. [dct:modified](https://w3c.github.io/dxwg/dcat/#Property:resource_update_date) - most recent date on which the item was changed, updated or modified
+9. [dct:creator](https://w3c.github.io/dxwg/dcat/#Property:resource_creator) - the author of the report
+10. [dct:publisher](https://w3c.github.io/dxwg/dcat/#Property:resource_publisher) - GSQ
+11. [dct:contactPoint](https://w3c.github.io/dxwg/dcat/#Property:resource_contact_point) - GSQ contact
+12. [prov:Agent](http://https://www.w3.org/TR/prov-o/#Agent)
+13. [rdfs:seeAlso](https://www.w3.org/TR/rdf-schema/#ch_seealso) - refers to secondary metadata
+14. [FOAF document](http://xmlns.com/foaf/spec/#term_Document) - a document with secondary metadata
+15. [time:properInterval](https://www.w3.org/TR/owl-time/#time:Interval)
+16. [time:hasStartTime](https://www.w3.org/TR/owl-time/#time:hasBeginning)
+17. [time:hasEndTime](https://www.w3.org/TR/owl-time/#time:hasBeginning)
+18. [locn:geometry](http://www.w3.org/ns/locn#geometry) - spatial coverage of the report expressed as lat/long, centroid, bounding box or simple polygon
+
+## [Distribution](https://w3c.github.io/dxwg/dcat/#Class:Distribution) properties not shown in the diagram:
+No distribution properties for Survey template
 
 ## Vocabularies
+1. [Survey relationship type](https://vocabs.gsq.digital/vocabulary/survey-relationship-type)
+2. [Survey type](https://vocabs.gsq.digital/vocabulary/survey-type)
+3. [Survey method](https://vocabs.gsq.digital/vocabulary/survey-method)
+
+## Seismic Survey
+Seismic survey is a specialised survey.
+
+<p align="center">
+<img src="model/seismic-survey.svg" width="700px"><br>  
+Figure 2: Seismic Survey profile</p
```

Comment:
- Model doesn't align with the survey-observation coupling discussed. Survey method is Ground, but the specific seismic sampling method now at observation level
- Mining Survey Status should just be survey Status
- Seismic dimensionality its its own special case that does not align with Standard BFO dimensionality